ui-tests/src/test/java/org/openmrs/reference/page/MarkPatientDeceasedPage.java

```diff
@@ -0,0 +1,127 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class MarkPatientDeceasedPage extends Page {
+	
+	private static final String URL = "/coreapps/markPatientDead.page";
+	private static final By DICTIONARY = By.id("dictionaryNavLink");
+	private static final By ADD_NEW_CONCEPT = By.xpath("//*[@id=\"content\"]/a[2]");
+	private static final By CONCEPT_NAME = By.id("namesByLocale[en].name");
+	private static final By ADD_ALERGY = By.xpath("//*[@id=\"codedDatatypeRow\"]/td/table/tbody/tr/td[2]/input[1]");
+	private static final By CAUSEBOX = By.id("newAnswerConcept");
+	private static final By FEVER = By.xpath("/html/body/ul[1]/li[2]/a");
+	private static final By ADDBUTTON = By.xpath("/html/body/div[5]/div[11]/button[1]");
+	private static final By SAVE = By.cssSelector("#saveDeleteButtons > input:nth-child(1)");
+	private static final By SEARCH = By.id("inputNode");
+	private static final By CONCEPT_SEARCH_MESSAGE = By.id("openmrsSearchTable_info");
+	private static final By COD = By.xpath("//*[@id=\"openmrsSearchTable\"]/tbody/tr[1]/td/span");
```

Comment:
Is this COD , or CODE? , it need to be more readable. Thanks.

Reply:
stands for Cause Of Death

Reply:
Thanks, @kdaud  and @sherrif10  let me work on the changes